### PR TITLE
Adding unit test for authentication module

### PR DESCRIPTION
**Pull request by rizperdana**
### What this PR does (required):
- This MR will adding test on `login` , `register`, `logout`, and `user`

### Any information needed to test this feature (required):
- Just run `npm run test` to run all available test case.

### Any issues with the current functionality (optional):
- As you see this test only cover basic test e.g: only testing functionality and making sure the result returned correctly according to expected response
- Next time i will add various corner cases like email already used or trying to hit protected endpoint

### CODE REVIEW

**Comment by rizperdana** [comment]
Current workspace CI doesn't work as intended thus my CI test failing, i will fix this in the future or may be switch to circle CI

File: server/db.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
 const mongoose = require("mongoose");
 
-const connectDB = async () => {
-  const conn = await mongoose.connect(process.env.MONGO_URI);
+const connectDB = async() => {
+    let db_uri
```
Comment (moffatethan) [inline server/db.js:4]: `dbUri` let's preference camelCase in JavaScript as it's the convention. Thanks!

**Review by moffatethan — CHANGES_REQUESTED** [review]
Seems to be some problem with the base of the branch. Please fix that along with the one comment. Thanks!
